### ISSUE TITLE
Documentation needs to reflect request options.data is now options.body

### ISSUE THREAD

**Issue by anderspitman** [MithrilJS/mithril.js#2444]
### Mithril Version: 2.0.0-rc.6

**Comment by isiahmeadows** [comment]
Duplicate of #2423

**Comment by anderspitman** [comment]
@isaaclyman the [2.0.0-rc.6 documentation](https://mithril.js.org/request.html) does not show this change. Am I missing something?

**Comment by isaaclyman** [comment]
Wrong number @anderspitman. I think you're looking for @isiahmeadows

**Comment by isiahmeadows** [comment]
@anderspitman It just hasn't been released yet. I've got about a dozen other things to fix before I release a new RC.

**Comment by isiahmeadows** [comment]
Also, our general policy is to close issues once a fix is merged into the main tree, not necessarily released. Just wanted to clarify that bit.

**Comment by anderspitman** [comment]
Sorry @isaaclyman!

@isiahmeadows I see. My confusion came from the fact that npm and the docs both show version 2.0.0-rc.6, but they're out of sync with each other. You're saying the fix is in master but not yet in the RC? That makes sense.

Somewhat off topic: is there a document anywhere that talks about the purpose of Mithril 2? I only switched because it's what the docs default to. Sounds like maybe I should go back to Mithril 1?

**Comment by isiahmeadows** [comment]
Mithril 2 is a *slightly* breaking update of Mithril v1, and it's what I'm pushing new people to use, because we want real-world experience. It's a release candidate, and I'm not adding any more breaking changes that aren't literal bug fixes.
